src/LifeCycle.hs

```diff
@@ -127,7 +127,7 @@ instance SOP.Generic Capability
 
 userCapabilities :: Role -> [Capability]
 userCapabilities = \case
-    Student    _clss -> []
+    Student    _clss -> [CanVote]
```

Comment:
ok, thanks.

this makes me think perhaps we should merge all `*Capabilities` functions into one after all that takes a context of super-type of all individual functions.  that would rule out issues like this.